__tests__/app.test.js

```diff
@@ -302,3 +302,32 @@ describe.only("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe.only("GET /api/users", () => {
+  test("200: Responds with an array of all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users.length).toBeGreaterThan(0);
+        users.forEach((user) => {
+          expect(user).toEqual({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("500: Responds with an error if database fails", () => {
+    jest
+      .spyOn(db, "query")
+      .mockRejectedValueOnce(new Error("Internal Server Error"));
+    return request(app)
+      .get("/api/users")
+      .expect(500)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Internal Server Error");
+      });
+  });
```

Comment:
This is not an appropriate test as we never want to actually reach a 500 error message. This represents that an error made it through all of our other catches which is not intended behaviour of this app